### DISCUSSION TITLE
Can't connect in Wifi mode: Firewall problem?

### DISCUSSION THREAD

**Discussion by ckl67** [RaspAP/raspap-webgui#1643] [Q&A] [answered]
Hello,

I can't connect with my Windows PC to my RASPAP in wifi mode, even though it works with a UBUNTU virtual machine on the same PC.

I have installed curl -sL https://install.raspap.com | bash

The problem must be with my Windows PC But I don't know how to get out of it
What's surprising is that it has already worked

Principle

I tried to take the issue on the origin, and found that the ping does not work from my Windows PC, but works with my Ubuntu VM.

![image](https://github.com/user-attachments/assets/c0c3bb2d-ea9b-43f5-b537-d1dae94e34fd)

What could explain this phenomenon?
**I haven't modified anything on my PC window, but the issue triggered up after a new installation of RASPAP**

I have tried to add a firewall rule but certainly I have missed some clue
Any idea ?

Thanks in advance

**Comment by ckl67** [comment]
Correction.. 

![image](https://github.com/user-attachments/assets/7474c923-1f19-4ed7-b9b9-73a9819a26ac)

**Comment by frankozland** [comment, marked as answer]
try resetting your iptables and restarting hotspot
sudo iptables -P INPUT ACCEPT
sudo iptables -P FORWARD ACCEPT
sudo iptables -P OUTPUT ACCEPT
sudo iptables -t nat -F
sudo iptables -t mangle -F
sudo iptables -F
sudo iptables -X

then try again.
https://www.digitalocean.com/community/tutorials/how-to-list-and-delete-iptables-firewall-rules

also - check your wifi channel and country code on raspap. 
to diagnose i sometimes drop to 802.11 to test connection.

log into raspap and see if you have outside access (ping www.google.com or ping 8.8.8.8)

run "ip a" on raspberry at command prompt and confirm the wireless device has a carrier an ip address and its up.

run iwconfig and make sure its showing up. it should look something like this:
```
@raspberrypi:~ $ iwconfig
lo        no wireless extensions.

eth0      no wireless extensions.

wlan0     IEEE 802.11  Mode:Master  Tx-Power=31 dBm   
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Power Management:on
          
wlan1     IEEE 802.11  ESSID:off/any  
          Mode:Managed  Access Point: Not-Associated   Tx-Power=20 dBm   
          Retry short limit:7   RTS thr=2347 B   Fragment thr:off
          Power Management:off
          
tun0      no wireless extensions.

**Reply by ckl67** [reply]
> try resetting your iptables and restarting hotspot sudo iptables -P INPUT ACCEPT sudo iptables -P FORWARD ACCEPT sudo iptables -P OUTPUT ACCEPT sudo iptables -t nat -F sudo iptables -t mangle -F sudo iptables -F sudo iptables -X
> 
> then try again. https://www.digitalocean.com/community/tutorials/how-to-list-and-delete-iptables-firewall-rules
> 
> also - check your wifi channel and country code on raspap. to diagnose i sometimes drop to 802.11 to test connection.
> 
> log into raspap and see if you have outside access (ping [www.google.com](http://www.google.com) or ping 8.8.8.8)
> 
> run "ip a" on raspberry at command prompt and confirm the wireless device has a carrier an ip address and its up.
> 
> run iwconfig and make sure its showing up. it should look something like this:
> 
> ```
> @raspberrypi:~ $ iwconfig
> lo        no wireless extensions.
> 
> eth0      no wireless extensions.
> 
> wlan0     IEEE 802.11  Mode:Master  Tx-Power=31 dBm   
>           Retry short limit:7   RTS thr:off   Fragment thr:off
>           Power Management:on
>           
> wlan1     IEEE 802.11  ESSID:off/any  
>           Mode:Managed  Access Point: Not-Associated   Tx-Power=20 dBm   
>           Retry short limit:7   RTS thr=2347 B   Fragment thr:off
>           Power Management:off
>           
> tun0      no wireless extensions.
> ```

Hi,
Thank you very much for your quick answer.
Unfortunately it didn't work
I've given up, as it works perfectly on an UBUNTU PC and on another WINDOWS PC 
There must be a problem with my firewall or proxy

Best Regards
Christian

**Comment by billz** [comment]
Disable your firewall and repeat the test.

Do you need a firewall? The vast majority of RaspAP installs are behind existing routers/firewalls. This is why RaspAP [doesn't install a firewall](https://github.com/RaspAP/raspap-webgui/discussions/1608) unless you add one as an [optional component](https://docs.raspap.com/firewall/).

**Reply by ckl67** [reply]
Unfortunately I cannot.... because it is my work laptop
Anywy it doesn't matter  I was just surprised that it has worked a while, and then no access 
Certainly a firewall or a Zscaller modification ...

**Reply by billz** [reply]
RaspAP's firewall is preconfigured to work with the services it uses, including optional ones like those for VPNs.

Since your firewall rules are blocking ping requests (ICMP) you could experiment by allowing them:

```
sudo iptables -A INPUT -p icmp --icmp-type 8/0 -j ACCEPT
sudo iptables -A INPUT -p icmp --icmp-type 0/0 -j ACCEPT
```

**Comment by ckl67** [comment]
Thank you for the hint, but I guess I will give up looking here.
I lost so many time to understand the behavior which suddenly appeared
